src/Base/Builder3D.cpp

```diff
@@ -69,10 +69,10 @@ Builder3D::~Builder3D()
 
 /**
  * Starts the definition of point set with the given point size and color.
- * If posible make not to much startPoints() and endPoints() calls. Try to put all
- * points in one set. 
+ * If possible don't make not to many startPoints() and endPoints() calls. 
```

Comment:
That's not correct either: don't make ... not ... to

Reply:
good catch. will fix and push updated commit.